main.go

```diff
@@ -148,15 +142,7 @@ func webdriver(w http.ResponseWriter, req *http.Request) {
 	if err != nil {
 		log.Printf("can't proxy webdriver requests to %v: %v", selenoidUri, err)
 	}
-
-	u.Path = req.URL.Path
-
-	proxy := &httputil.ReverseProxy{
-		Director: func(r *http.Request) {
-			r.URL = u
-		},
-	}
-	proxy.ServeHTTP(w, req)
+	httputil.NewSingleHostReverseProxy(u).ServeHTTP(w, req)
```

Comment:
Will not work. In our code we proxy like this:
```
selenoid-ui:/wd/hub -> selenoid:/wd/hub
```
With this change it will proxy like the following:
```
selenoid-ui:/wd/hub -> selenoid:/wd/hub/wd/hub
```

Reply:
hmm, but it worked for me, will write a test